homeassistant/components/pglab/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for Tasmota."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.service_info.mqtt import MqttServiceInfo, ReceivePayloadType
+
+from .const import CONF_DISCOVERY_PREFIX, DISCOVERY_TOPIC, DOMAIN
+
+
+class FlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize flow."""
+        self._prefix = DISCOVERY_TOPIC
+        self._payload: ReceivePayloadType = ""
+
+    async def async_step_mqtt(self, discovery_info: MqttServiceInfo) -> FlowResult:
+        """Handle a flow initialized by MQTT discovery."""
+        if self._async_in_progress() or self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        await self.async_set_unique_id(DOMAIN)
+
+        # Validate the message, abort if it fails
+        if not discovery_info.topic.endswith("/config"):
+            # Not a PG LAB Electronics discovery message
+            return self.async_abort(reason="invalid_discovery_info")
+        if not discovery_info.payload:
+            # Empty payload, unexpesd payload
```

Comment:
```suggestion
            # Empty payload, unexpected payload
```

Reply:
Thank you for spotting the spelling mistake. I'll change it

Reply:
I think you forgot